src/main/java/filestorage/Storage.java

```diff
@@ -0,0 +1,62 @@
+package filestorage;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.List;
+
+import exceptions.TaylorException;
+import tasks.Task;
+
+/**
+ * To read data from and save data into Hard Disk
+ */
+public class Storage {
+    /**
+     * No constructor of class needed
+     */
+    private Storage() {
+        throw new AssertionError("Constructor is not allowed");
+    }
+
+    /**
+     * Read File from Hard Disk
```

Comment:
Perhaps, as stated in the coding standards, your method header comments can start in the form Reads, ... Returns ..., Sends ..., Adds ... etc instead of Read or Reading?

Reply:
Thank you for highlighting something I have missed out on!